### PR TITLE
Updated AboutUs.md and added portfolios for each member

### LINKED ISSUE #5: Include introduction in About Us Page

### PR DESCRIPTION
**Pull request by JuliaPoo**
Updated according to [week 7 requirements](https://nus-cs2103-ay2324s2.github.io/website/schedule/week7/project.html).
It is not completed, we require other members to add their profile pictures. Once this is done, we can close #5.

I've also added the portfolio files to log each member's contributions (as templated by `team/johndoe.md`).

### CODE REVIEW

File: docs/team/juliapoo.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+  layout: default.md
+  title: "Jule's Project Portfolio Page"
+---
+
+### Project: CCA Manager
+
+CCA Manager is a **desktop app for managing contacts, optimized for use via a  Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, CCA Manager can get your contact management tasks done faster than traditional GUI apps.
```
Comment (Fidget-Spinner) [inline docs/team/juliapoo.md:8]: Just checking, do the files need to be capitalised according to our GH usernames, or is all lower case fine? Or maybe it doesn't matter at all?

File: docs/AboutUs.md [hunk 2]
```diff
@@ -11,51 +11,53 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
 <img src="images/johndoe.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/Fidget-Spinner/)]
+[[portfolio](team/fidget-spinner.md)]
 
-* Role: Project Advisor
+* Role: Team Lead
```
Comment (Fidget-Spinner) [inline docs/AboutUs.md:21]: You can put me as developer as well. Apparently an all-developer team with no leader is an actual team style in CS2103T material.

**Review by Fidget-Spinner — COMMENTED** [review]
Looks good overall, just two minor comments.

File: docs/team/juliapoo.md (hunk 1 above)
Comment (JuliaPoo) [reply docs/team/juliapoo.md:8]: Nothing is stated about the portfolio files, but the profile pictures are required to be lower cased versions of our GH usernames.

File: docs/AboutUs.md (hunk 2 above)
Comment (JuliaPoo) [reply docs/AboutUs.md:21]: Okay I'll do that

**Review by Fidget-Spinner — APPROVED** [review]

**Comment by Fidget-Spinner** [comment]
Note: IIRC CS2103T commits need to be in present tense so Update not Updated.

**Pull request merged**